actions/class.Items.php

```diff
@@ -83,24 +83,6 @@ protected function getClassService()
         return $this->service;
     }
 
-    /**
```

Comment:
Do you know why was it need to override?

Maybe to set specific permissions just for items before? Are you sure it wont break previous behavior?